packages/plop/src/plop.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
+import "tsx";
```

Comment:
This is currently necessary, there is something strange going on with the test runner where the normal `NODE_*` env vars are not getting through properly.